apps/daimo-mobile/src/view/screen/DeviceScreen.tsx

```diff
@@ -108,7 +108,7 @@ export function DeviceScreen({ route, navigation }: Props) {
 
   useEffect(() => {
     (async () => {
-      if (status !== "success" || !account) return;
+      if (status !== "loading" || !account) return;
```

Comment:
was this just a bug before?

Reply:
https://github.com/daimo-eth/daimo/pull/806#discussion_r1515549117

Reply:
no, both are fine but if our paymaster is down remove device doesn't work. Reverted 👍